stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/StreamFileUtil.kt

```diff
@@ -218,10 +217,12 @@ public object StreamFileUtil {
 
                 if (response.isSuccess) {
                     // write the response to a file
-                    val byteArray = response.data().byteStream().readBytes()
-                    val fileOutputStream = FileOutputStream(file)
-                    fileOutputStream.write(byteArray)
-                    fileOutputStream.close()
+                    response.data().byteStream().use { inputStream ->
```

Comment:
What do you think about using `use` for output stream as well:

```
response.data().byteStream().use { inputStream ->
    file.outputStream().use { outputStream->
        inputStream.copyTo(outputStream)
    }
}
```

Reply:
Implemented the suggested solution.